### PR TITLE
Daily spellbook sub project

### PR DESCRIPTION
**Pull request by aalan3**
This PR initiates the a new sub project for models that are going to be run daily. 

I also added the necessary CI steps,and (generated and) uploaded a manifest to S3 so that CI works.

### CODE REVIEW

File: daily_spellbook/packages.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+packages:
+  - package: dbt-labs/dbt_utils
+    version: 1.1.1
+  - package: dbt-labs/codegen
+    version: 0.12.1
+  - package: starburstdata/trino_utils
+    version: 0.6.0
```
Comment (0xRobin) [inline daily_spellbook/packages.yml:7]: these are new? Should we also add them to the main `packages.yml`?

Comment (aalan3) [reply daily_spellbook/packages.yml:7]: Yes probably, @jeff-dude added these files but we can update main in a separate PR

Comment (jeff-dude) [reply daily_spellbook/packages.yml:7]: these were added during @couralex6 discovery of subprojects during tokens build. i believe codegen was intended to help the process of moving files around, but not actively used? then trino_utils is just trino-specific dbt_utils. we can just add per subproject as needed. as of now, i'm not sure we actually use any from there

Comment (jeff-dude) [reply daily_spellbook/packages.yml:7]: maybe i'll remove from this new subproject, to ensure it doesn't break anything without

File: daily_spellbook/dbt_project.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      ethereum:
+        +schema: chainlink_ethereum
+      fantom:
+        +schema: chainlink_fantom
+      gnosis:
+        +schema: chainlink_gnosis
+      optimism:
+        +schema: chainlink_optimism
+      polygon:
+        +schema: chainlink_polygon
```
Comment (jeff-dude) [inline daily_spellbook/dbt_project.yml:91]: just noting that we will want to remove these soon and put in models. just a lot of models to do. we can wait until the team refactors the code into macros (already underway)

Comment (aalan3) [reply daily_spellbook/dbt_project.yml:91]: Yeah agree, was surprised to see these in the project file

**Comment by jeff-dude** [comment]
i'm going to push a commit to try to remove those extra packages for now, to prove they aren't needed. we're also in a tough spot here, where CI won't run because the changes to CI are in the same PR.

**Comment by jeff-dude** [comment]
> i'm going to push a commit to try to remove those extra packages for now, to prove they aren't needed. we're also in a tough spot here, where CI won't run because the changes to CI are in the same PR.

actually, looks like the workflow does run, but no models kicked off?

**Comment by jeff-dude** [comment]
mentioned in DMs with alan, but we should double check the downstream impacts of moving these. for instance, what else in spellbook reads from any of these as a `ref()`

**Pull request merged**